synapse/handlers/e2e_room_keys.py

```diff
@@ -207,6 +207,13 @@ def upload_room_keys(self, user_id, version, room_keys):
             changed = False  # if anything has changed, we need to update the etag
             for room_id, room in iteritems(room_keys["rooms"]):
                 for session_id, room_key in iteritems(room["sessions"]):
+                    if not isinstance(room_key["is_verified"], bool):
+                        msg = (
+                            "is_verified must be a boolean in keys for room %s"
+                            % room_id
```

Comment:
Not sure it is necessary, but looks like a `room_id` could have multiple `session_id` entries in it, might be worth including in the error message.

Reply:
Done in https://github.com/matrix-org/synapse/pull/7045/commits/297aaf48166f153d35c38160d0c747770d925f39